app/views/short_url_requests/_form.html.erb

```diff
@@ -54,4 +44,33 @@
       <% end %>
     </div>
   </fieldset>
+
+  <fieldset>
+    <h2>Advanced options</h2>
+    <p>Leave as defaults if unsure.</p>
+    <div class="form-group">
+      <%= f.label 'From redirects' %>
+      <%= f.select :route_type, options_for_select({ "Exact URL only" => "exact", "URL and all pages under it" => "prefix" }, @short_url_request.route_type), {}, class: 'form-control input-md-6' %>
+      <p>Do not redirect all pages unless you need any page with an address which starts with your input to also be redirected. For example, if you add <strong>gov.uk/address/</strong> then <strong>gov.uk/address/gone</strong> would also be redirected.</p>
+    </div>
+
+    <div class="form-group">
+      <%= f.label 'Segments' %>
+      <%= f.select :segments_mode, options_for_select({ "Discard" => "ignore", "Keep" => "preserve" }, @short_url_request.segments_mode), {}, class: 'form-control input-md-6' %>
+    </div>
+
+    <h3>Help with Segments</h3>
+    <h4>Discard</h4>
+    <h5>Using "Exact URL only"</h5>
+    <p>For source <strong>/from?q=123</strong> and target url <strong>/target</strong> will redirect to <strong>/target</strong></p>
+    <h5>Using "URL and all pages under it"</h5>
+    <p>For source <strong>/from?q=123</strong> and target url <strong>/target</strong> will redirect to <strong>/target</strong> and also <strong>/from/doe/a/deer</strong> will redirect to <strong>/target</strong></p>
+
+
+    <h4>Keep</h4>
+    <h5>Using "Exact URL only"</h5>
+    <p>For source <strong>/from/?q=123</strong> and target url <strong>/target</strong> will redirect to <strong>/target?q=123</strong></p>
+    <h5>Using "URL and all pages under it"</h5>
+    <p>For source <strong>/from/?q=123</strong> and target url <strong>/target</strong> will redirect to <strong>/target/page/?q=123</strong> and also <strong>/from/doe/a/deer</strong> will redirect to <strong>/target/doe/a/deer</strong></p>
+  </fieldset>
```

Comment:
This documentation is looking really good Kelvin :) Did you consider splitting out the treatment of the query parameters? As in, say that when you select to preserve the segments when redirecting a url and all pages under it, any path suffix will be included in the URL the user is redirect to, and when you choose ignore, then the exact target specified will be used. Then also say that query parameters are only preserved if preserve is selected, and discarded if ignore is selected.